packages/connector-mock-email-alternative/package.json

```diff
@@ -1,48 +1,62 @@
 {
-  "name": "@logto/connector-mock-email-alternative",
-  "version": "1.0.0-beta.12",
-  "description": "Mock Standard Email Service connector implementation for integration tests only.",
   "main": "./lib/index.js",
   "exports": "./lib/index.js",
-  "author": "Silverhand Inc. <contact@silverhand.io>",
   "license": "MPL-2.0",
   "files": [
     "lib",
     "docs",
     "logo.svg",
-    "README.md"
+    "logo-dark.svg"
   ],
   "scripts": {
+    "package:sync": "cd ../../ && pnpm package:sync",
     "precommit": "lint-staged",
     "build": "rm -rf lib/ && ncc build src/index.ts -o lib",
     "dev": "tsc -p tsconfig.build.json --watch --preserveWatchOutput --incremental",
     "lint": "eslint --ext .ts src",
     "lint:report": "pnpm lint --format json --output-file report.json",
+    "test": "echo No test available; exit 0;",
+    "test:coverage": "pnpm run test --coverage --silent",
     "prepack": "pnpm build"
   },
   "dependencies": {
-    "@logto/connector-kit": "^1.0.0-beta.26",
+    "@logto/connector-kit": "1.0.0-beta.29",
     "@silverhand/essentials": "^1.2.0",
-    "zod": "^3.14.3"
+    "got": "^11.8.2",
+    "snakecase-keys": "^5.1.0",
+    "zod": "^3.20.2"
   },
   "devDependencies": {
+    "iconv-lite": "0.6.3",
+    "@jest/types": "^29.3.1",
     "@silverhand/eslint-config": "1.3.0",
     "@silverhand/ts-config": "1.2.1",
+    "@silverhand/jest-config": "1.2.2",
+    "@types/jest": "^29.2.4",
     "@types/node": "^16.3.1",
     "@vercel/ncc": "^0.36.0",
+    "@types/supertest": "^2.0.11",
```

Comment:
Why not using package.extend.json for this one?

Reply:
rebase error